MANIFEST.in

```diff
@@ -0,0 +1,86 @@
+include pybamm/input/parameters/lithium-ion/cathodes/lico2_Marquis2019/lico2_entropic_change_Moura2016.py
+include pybamm/input/parameters/lithium-ion/cathodes/lico2_Marquis2019/lico2_diffusivity_Dualfoil1998.py
+include pybamm/input/parameters/lithium-ion/electrolytes/lipf6_Marquis2019/electrolyte_diffusivity_Capiglia1999.py
+include pybamm/input/parameters/lithium-ion/electrolytes/lipf6_Marquis2019/electrolyte_conductivity_Capiglia1999.py
+include pybamm/input/parameters/lithium-ion/electrolytes/lipf6_Kim2011/electrolyte_diffusivity_Kim2011.py
+include pybamm/input/parameters/lithium-ion/electrolytes/lipf6_Kim2011/electrolyte_conductivity_Kim2011.py
+include pybamm/input/parameters/lithium-ion/electrolytes/lipf6_Nyman2008/electrolyte_conductivity_Nyman2008.py
+include pybamm/input/parameters/lithium-ion/electrolytes/lipf6_Nyman2008/electrolyte_diffusivity_Nyman2008.py
+include pybamm/input/parameters/lead-acid/anodes/lead_Sulzer2019/README.md
+include pybamm/input/parameters/lead-acid/cells/BBOXX_Sulzer2019/README.md
+include pybamm/input/parameters/lead-acid/cathodes/lead_dioxide_Sulzer2019/README.md
+include pybamm/input/parameters/lead-acid/experiments/1C_discharge_from_full/README.md
+include pybamm/input/parameters/lead-acid/electrolytes/sulfuric_acid_Sulzer2019/README.md
+include pybamm/input/parameters/lead-acid/separators/agm_Sulzer2019/README.md
+include pybamm/input/parameters/lithium-ion/anodes/graphite_Chen2020/README.md
+include pybamm/input/parameters/lithium-ion/anodes/graphite_mcmb2528_Marquis2019/README.md
+include pybamm/input/parameters/lithium-ion/anodes/graphite_Kim2011/README.md
+include pybamm/input/parameters/lithium-ion/cells/Kim2011/README.md
+include pybamm/input/parameters/lithium-ion/cells/kokam_Marquis2019/README.md
+include pybamm/input/parameters/lithium-ion/cells/LGM50_Chen2020/README.md
+include pybamm/input/parameters/lithium-ion/cathodes/nca_Kim2011/README.md
+include pybamm/input/parameters/lithium-ion/cathodes/nmc_Chen2020/README.md
+include pybamm/input/parameters/lithium-ion/cathodes/lico2_Marquis2019/README.md
+include pybamm/input/parameters/lithium-ion/experiments/1C_discharge_from_full_Marquis2019/README.md
+include pybamm/input/parameters/lithium-ion/experiments/1C_discharge_from_full_Kim2011/README.md
+include pybamm/input/parameters/lithium-ion/experiments/1C_discharge_from_full_Chen2020/README.md
+include pybamm/input/parameters/lithium-ion/electrolytes/lipf6_Marquis2019/README.md
+include pybamm/input/parameters/lithium-ion/electrolytes/lipf6_Kim2011/README.md
+include pybamm/input/parameters/lithium-ion/electrolytes/lipf6_Nyman2008/README.md
+include pybamm/input/parameters/lithium-ion/separators/separator_Kim2011/README.md
+include pybamm/input/parameters/lithium-ion/separators/separator_Chen2020/README.md
+include pybamm/input/parameters/lithium-ion/separators/separator_Marquis2019/README.md
+include pybamm/version
+include pybamm/CITATIONS.txt
```

Comment:
do new parameter sets automatically get added to this, or does it need to be done manually?

Reply:
It's manual at the moment, and I have a script that goes through the directories and writes the `MANIFEST.in`.
This file is actually only required for including the parameters in the source distribution (the archive uploaded ti PyPI) and not in the built distribution (the wheel).
The data included in the wheel is defined in the setup.py (variable `pybamm_data`) and everything inside `input/` is included.

This is not good practice though, as python package distributions are meant to only ship the source code... As the number of parameters grow we should probably think of other ways to distribute parameters to users. Another repo? I'm planning to write an issue about this.